keps/sig-api-machinery/1040-priority-and-fairness/README.md

```diff
@@ -717,9 +717,11 @@ mu(i,t) = min(rho(i,t), mu_fair(t))
 
 where:
 - `i` identifies a queue,
-- `rho(i,t)` is the rate requested by queue `i` at time `t` and is defined to be
-  the product of `mu_single` and the number of packets of that queue
-  that are unsent in the virtual world at time `t`,
+- `rho(i,t)` is the rate requested by queue `i` at time `t` and is
+  defined to be the product of `mu_single` and the number of packets
+  of that queue that are not fully sent in the virtual world at time
+  `t` (those for which `t_arrive(i,j) <= t` and whose transmisison
```

Comment:
nit: transmission

[this is what is failing verify scripts]

Reply:
Thanks for the catch.

Reply:
fixed